changedetectionio/api/Notifications.py

```diff
@@ -0,0 +1,86 @@
+from flask_restful import Resource
+from . import auth
+from flask_restful import abort, Resource
+from flask import request
+from . import auth
+
+class Notifications(Resource):
```

Comment:
This endpoint is more about just adding a notification URL, and not the same as the UI

Could you call this something like notificationsSingular

and then add the main endpoint called `notifications` where one can submit a whole list (same like in UI)



also this needs to work with Groups aswell, is there a way to make this update a group notifications?

Reply:
That way a list of notifications could be added - in the case that they want to reset the list you would just send an empty list